applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -12,7 +20,7 @@
         @fragments.commercial.adSlot(
             slotName,
             Seq("gallery-inline", "dark") ++ (if(isMobile) Some("mobile") else None),
-            Map("mobile" -> sizes),
+            Map("mobile" -> (sizes ++ additionalSizes)),
```

Comment:
You should add `additionalSizes` to the `desktop` (or even `leftCol` or `wide`) breakpoint instead, so: `Map("mobile" -> sizes) + additionalSizes`